CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - An errant space in the windows commands to run monkey manually. #1153
 - gevent tracebacks in console output. #859
 - Crash and failure to run PBAs if max depth reached. #1374
+- Briefly see "Getting Started" page before the landing page after registration. #1399
```

Comment:
We don't need a changelog entry for this. This bug was introduced in this release, so this bugfix is not a change since the last release.